spaceship/lib/spaceship/portal/ui/select_team.rb

```
@@ -32,9 +32,15 @@ class UserInterface
       #   ]
 
       def select_team
-        teams = client.teams
-
-        raise "Your account is in no teams" if teams.count == 0
+        begin
+          teams = client.teams
+          raise "Your account is in no teams" if teams.count == 0
```

Why do we first raise the error, just to catch it and swallow it?